pillar.example

```diff
@@ -1,28 +1,11 @@
 chrony:
   config:
```

Comment:
the config could be removed. it should only be
```
chrony:
  ntpservers:
```

Reply:
Sorry I got mixed up with samba formula. fixing.